packages/office-addin-sso/src/configure.ts

```diff
@@ -38,8 +44,22 @@ export async function configureSSOApplication(manifestPath: string, port: string
         await logoutAzure();
 
         console.log(chalk.green(`Application with id ${applicationJson['appId']} successfully registered in Azure.  Go to https://ms.portal.azure.com/#home and search for 'App Registrations' to see your application`));
+
+        // Log end time for configuration process and compute in seconds
+        const endSsoCongiguration = (new Date()).getTime();
+        const durationSsoConfiguration = (endSsoCongiguration - startSsoCongiguration) /1000
+        
+        // Send usage data
+        usageDataInfo = {
+            Method: ['configureSSOApplication'],
+            configDuration: [durationSsoConfiguration],
+            Platform: [process.platform],
+            Succeeded: [true]
+        }
+        usageDataHelper.sendUsageDataCustomEvent(usageDataInfo);
     }
     else {
+        usageDataHelper.sendUsageDataException('configureSSOApplication', 'Login to Azure did not succeed');
```

Comment:
I see many examples here were it sends the usage data exception and then throws it. To ensure that the message matches between the two, it would be better to first assign the message to a variable and use the same variable for both.  This applies to all of the places where you report the usage data exception.

```
const errorMessage = "Unable to log in to Azure.";
usageDataHelper.sendUsageDataException('configureSSOApplication`, errorMessage);
throw new Error(errorMessage);
```

Reply:
Good idea. Fixed throughout